engine/src/flutter/lib/ui/window.dart

```diff
@@ -981,7 +981,7 @@ class AccessibilityFeatures {
   /// Use this flag to conditionally avoid making announcements on Android.
   // This index check is inverted (== 0 vs != 0); far more platforms support
   // "announce" than discourage it.
-  bool get announce => _kNoAnnounceIndex & _index == 0;
+  bool get supportsAnnounce => _kNoAnnounceIndex & _index == 0;
```

Comment:
The double-negative here - supports is true if the no announce bit is not set - is not recommended by our style guide: https://github.com/flutter/flutter/blob/master/docs/contributing/Style-guide-for-Flutter-repo.md#avoid-double-negatives-in-apis

IMO this would be clearer if we made this bit `kSupportsAnnounceIndex` and then checked that the bit is set. Is it too late to change that?

Reply:
@ash2moon this pr has had too many separate different forms that I cant find the original conversation where we decided to use the negative in the engine and the positive in the public api. 

Can you please link that discussion here? 

@loic-sharma yes we know and the decision was made to avoid having the wrong value on embedders not under our control (and less code in the embedders we do) since false is the correct default for most platforms.